tests/bluetooth/host/common/defaults_cmake.txt

```diff
@@ -0,0 +1,55 @@
+#
+# Common include directories and source files for bluetooth unit tests
+#
+
+include_directories(
+  ${CMAKE_CURRENT_SOURCE_DIR}/
+  ${CMAKE_CURRENT_SOURCE_DIR}/src
+  ${ZEPHYR_BASE}/tests/bluetooth/host/mock_host/include
+  ${ZEPHYR_BASE}/tests/bluetooth/host/common/include
+  ${ZEPHYR_BASE}/include/zephyr/net
+  ${ZEPHYR_BASE}/include/zephyr/bluetooth
+  ${ZEPHYR_BASE}/subsys/bluetooth
+  ${ZEPHYR_BASE}/subsys/bluetooth/host
+  ${ZEPHYR_BASE}/subsys/testsuite/ztest/include/zephyr
+)
+
+FILE(GLOB host_sources
+  ${ZEPHYR_BASE}/subsys/bluetooth/host/buf.c
+)
+
+FILE(GLOB mock_sources
+  ${ZEPHYR_BASE}/tests/bluetooth/host/mock_host/src/net/buf.c
+  ${ZEPHYR_BASE}/tests/bluetooth/host/mock_host/src/iso.c
+  ${ZEPHYR_BASE}/tests/bluetooth/host/mock_host/src/hci_core.c
+  ${ZEPHYR_BASE}/tests/bluetooth/host/mock_host/src/assert.c
+)
```

Comment:
This is also not common to `test/bluetooth/host`. (If we want to test `hci_core.c`, we don't want to mock it.)